src/components/Explore/utils.ts

```diff
@@ -366,6 +385,53 @@ export const exploreMetricData: {
       },
     ],
   },
+
+  // Todo (update with real content)
```

Comment:
Maybe we can start using `TODO(8.2)` to make them easier to find later? I can comb the feature branch to find 8.2 TODOs and update the comments

Reply:
good idea